tool/update_tars.dart

```diff
@@ -20,4 +22,29 @@ void main() {
           '${getCronetName(platform)} not found in .dart_tool/cronet/$platform');
     }
   }
+  // Make tarball for android releases.
+  final android =
+      '${findPackageRoot().toFilePath(windows: Platform.isWindows)}/android';
+  if (!Directory('$android/libs').existsSync()) return;
+  Directory('android/libs').createSync(recursive: true);
```

Comment:
You can consider creating constants for paths like these. Since they are being used at a lot of places. If you do, make sure you give those constants a relevant name.